### DISCUSSION TITLE
Integrate drawio in proprietary software

### DISCUSSION THREAD

**Discussion by MuratPlt** [jgraph/drawio#4227] [Q&A]
Hey all

I just want to make sure to I do not do anything wrong.
Do I understand correctly that the Apache license makes it possible to embed drawio in commercially used software at no cost?
So, for example, I can redirect my users from my software to "embed.diagrams.net" and save the diagram produced in my software.
Or run drawio myself in a Docker environment and redirect my users there (?).

If the former is also possible, I was wondering how the production of the "embed.diagrams.net" server is financed when it's free.

Thank you!

**Comment by davidjgraph** [comment]
> Do I understand correctly that the Apache license makes it possible to embed drawio in commercially used software at no cost?

https://github.com/jgraph/drawio/discussions/3805

You can use embed or run your own version. Embed is just a static delivered site, there's no backend.

> how the production of the "embed.diagrams.net" server is financed when it's free.

Because it's a static site served over CDN, the cost is pretty much zero.
